games/migrations/0013_auto_20190718_1736.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.2 on 2019-07-18 17:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('games', '0012_adventure_tier'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
What is this migration for? Should it be actually part of this PR?

Reply:
It was migration created automaticly by django when i run makemigrations. Maybe it's becouse of sqllite... As i can see this is copy of 0012_adventure_tier, so i will remove this